index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title></title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;300&family=Playfair+Display:wght@400;500&family=Poppins:wght@100;300&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="css/styles.css">
+
+</head>
+
+<body>
+  <!-- Header section -->
+  <header class="header">
+    <ul class="nav-bar">
+      <li class="nav-link"><a href="#" class="logo">Welcome</a></li>
+      <li class="nav-link">
+        <a href="#">
+          <img src="images/menubar.svg" width="24px" height="24px" alt="">
+        </a>
+      </li>
+    </ul>
+  </header>
```

Comment:
Good job using a `header` tag here, but I suggest using a `nav` tag instead of a `ul` tag to wrap your logo and hamburger button, this will make your HTML page more semantically correct 😄